.github/workflows/lint-testunit.yml

```
@@ -0,0 +1,44 @@
+name: Lint and Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint-testunit:
+    if: ${{ github.repository == 'RocketChat/Rocket.Chat.ReactNative' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
```

If this works, it's going to help a lot on the long term, since it won't require us to bump it manually from time to time 🤞🏼 

```suggestion
          node-version: 'lts/*'
```